miso-web/src/main/webapp/scripts/warning.js

```diff
@@ -22,38 +22,63 @@
  */
 
 var WarningTarget = {};
-var Warning = {
-  generateHeaderWarnings: function(warnings){
-    if(warnings.length == 0){
-      return;
+var Warning = (function($) {
+
+  return {
+    generateHeaderWarnings: function(containerId, target, item) {
+      // Note: header warnings currently ignore the warning level and show all warnings in red
+      var warnings = getWarnings(target, item);
+      if (warnings.length == 0) {
+        return;
+      }
+      var container = $('#' + containerId);
+      container.append($('<span>').css('float', 'right').append($('<img>').attr('src', '/styles/images/fail.png')));
```

Comment:
These pass/fail icons are the one place where we use red & green for contrast, and they're pretty identical for users with red/green colourblindness. Could alt text be added to give additional cues?

Reply:
The pool selection dialog now includes red icon with alt text "Warnings" for ones with warnings and no icon for ones without